lib/stdlib.ngs

```diff
@@ -2105,6 +2105,22 @@ section "Range" {
 	TEST 1 =~ 0..5
 	TEST 0 =~ 0..5
 	TEST 5 !~ 0..5
+
+	doc Ensure that n is in the range. 
+	doc n in range returns n
+	doc %EX - ensure(1, 3..10) #3
+	F ensure(n:Int, r:NumRange){
+
+    	if n in r {
```

Comment:
Multiple chained if .. else clauses should be `cond` instead:

```
cond {
  COND_EXPR_1 RESULT_EXPR_1
  ...
  COND_EXPR_N RESULT_EXPR_N
}
```

RESULT_EXPR can also be `{ some; code; }`

Reply:
This is interesting, I did not know about it. Thanks

Reply:
for this case I do not think I need it.

Reply:
> This is interesting, I did not know about it. Thanks

Yep... https://github.com/ngs-lang/ngs/issues/592

Reply:
> for this case I do not think I need it.

You don't have to but it's a code quality issue